feature-libs/product-configurator/rulebased/components/attribute/quantity/configurator-attribute-quantity.service.ts

```diff
@@ -45,4 +76,20 @@ export class ConfiguratorAttributeQuantityService {
         return false;
     }
   }
+
+  /**
+   * Checks if the zero quantity is allowed
+   *
+   * @param attribute Configurator Attribute
```

Comment:
According to JSDoc we could add a type of the corresponding parameter, for instance:
`param {Configurator.Attribute} attribute - Configurator attribute`

Reply:
done